data/json/items/comestibles.json

```diff
@@ -8483,7 +8606,7 @@
         "id" : "syrup",
         "name" : "maple syrup",
         "name_plural" : "maple syrup",
-        "weight" : 28,
+        "weight" : 334,
```

Comment:
Yes, that was checked. Maple syrup is based on water and water is 250g per 250ml (1 volume). So it has to be at least 250g. That nets a 84g for the added sugar and mineral which is ok based on my research.